client/components/EventsListContainer/index.jsx

```diff
@@ -75,7 +75,8 @@ class EventsListComponent extends React.Component {
                                 loadMoreEvents={this.props.loadMoreEvents}
                                 selectedEvents={this.props.selectedEvents}
                                 onEventSelectChange={this.props.onEventSelectChange}
-                                privileges={privileges} />
+                                privileges={privileges}
+                                session={session} />
```

Comment:
It is needed to pass on EventList as a prop - as EventList is not connected to the store. Instead of connecting EventList to store only for this purpuse, I thought it is better to send 'session' as a prop. EventList needs to know it so that it can compute if an EventItem is locked in this session, in which case, the owner of the lock can still go on and spike or unspike an event. Just as in Superdesk authoring where the current owner of the lock can spike an item he/she is editing.